docs/pt/docs/tutorial/request-forms.md

```diff
@@ -0,0 +1,58 @@
+# Dados de Formulário
+
+Quando você precisar receber campos de um formulário ao invés de JSON, você pode usar o `Form`.
+
+!!! info "Informação"
+    Para usar o `Form`, primeiro instale <a href="https://andrew-d.github.io/python-multipart/" class="external-link" target="_blank">`python-multipart`</a>.
+
+    E.g. `pip install python-multipart`.
+
+## Importando `Form`
+
+Importando `Form` pela `fastapi`:
```

Comment:
Maybe "por" instead of "pela"? 🤔

Reply:
Hey Kludex! In the portuguese language we have gender differentiation in some words, for example:

The word _library_ is translated to _biblioteca_, which is a female word (it ends with an "a"), that's why we use _pela_ instead of _por_, because _pela_ is a female version of _por_, got it?

If you still want me to use "por", I will use it, no problems! I tried to explain how our language works.

I also have a question, what about the Build Docs Test? Mine was cancelled

Reply:
Eu sou brasileiro... 👀 Por que eu iria sugerir algo sem falar a língua? 😅 

> The word library is translated to biblioteca, which is a female word (it ends with an "a"), that's why we use pela instead of por, because pela is a female version of por, got it?

Mas "pacote" eh masculino. 👀 E... Nem toda palavra que termina com A eh feminina. 😗

Meu ponto nao eh nem esse, eh que no caso vc fez concordancia com a ideia (biblioteca), que no caso seria uma [silepse](https://mundoeducacao.uol.com.br/gramatica/silepse-concordancia.htm), e nao com FastAPI.

Anyway, ja na _home_ traduzida, da pra ver que a gente usa FastAPI no masculino: https://fastapi.tiangolo.com/pt/ 👀

Reply:
Ah, e nao tem problema em usar silepses. 🙏

Reply:
Na minha opnião seria muito mais interessante o uso de `através` do que a utilização de por e pela nesse contexto, o que acham?